blockpage_signatures.json

```diff
@@ -451,131 +451,186 @@
+{"fingerprint":"c_isp_us_virginia_edu_satellite","pattern":"has been blocked by <a href=\"http://security\\.virginia\\.edu\">University Information Security</a> for hosting phishing, malware, or other malicious behavior\\."}
+{"fingerprint":"d_corp_ca_cira_1_satellite","pattern":"Location: https://cira\\.ca:443/CanadianShield/Active/FamilyFilter"}
+{"fingerprint":"d_corp_ca_cira_2_satellite","pattern":"Location: https://cira\\.ca:443/CanadianShield/Active/ScamFilter"}
+{"fingerprint":"d_corp_ca_cira_3_satellite","pattern":"Location: https://www\\.cira\\.ca:443/CanadianShield/Active/PhishingBlock"}
+{"fingerprint":"d_corp_co_mercanet_satellite","pattern":"Location: http://www\\.merca\\.net\\.co/mintic"}
+{"fingerprint":"d_corp_gr_opi_redirect_302_satellite","pattern":" <a href=\"https://opi\\.gr/edppi_block/edppi_block\\.html\""}
+{"fingerprint":"d_corp_id_jiipe_satellite","pattern":"<title>Jiipe \\| Internet Positive</title>"}
+{"fingerprint":"d_corp_microsoft_1_satellite","pattern":"<h4>You have been redirected from a malicious site</h4>"}
+{"fingerprint":"d_corp_microsoft_2_satellite","pattern":"<h4>Please use our domain review process if you feel this site is legitimate or should be in another category\\.</h4>"}
+{"fingerprint":"d_corp_multacom_satellite","pattern":"<title>Page Restricted</title>\r"}
+{"fingerprint":"d_corp_ru_skydns_satellite","pattern":" <base href=\"fhttps://cdn-b\\.skydns\\.ru/static/\" target=\"_blank\">"}
+{"fingerprint":"d_corp_zyxel_satellite","pattern":"<title>Zyxel Security Cloud-DNS Filter Service Portal</title>"}
+{"fingerprint":"e_unk_do_sengkecloud_satellite","pattern":"Location: https://www\\.sengkedcloud\\.com/"}
+{"fingerprint":"e_unk_gr_redirect_302_satellite","pattern":"Location: http://nba\\.gov\\.cy/"}
+{"fingerprint":"e_unk_gr_satellite","pattern":"<meta http-equiv=\"refresh\" content=\"2; url=http://nba\\.gov\\.cy/\" />"}
+{"fingerprint":"e_unk_id_2_satellite","pattern":"Location: /blocked"}
+{"fingerprint":"e_unk_id_satellite","pattern":"<title>DNS Bersih Baraya</title>"}
+{"fingerprint":"e_unk_pwlan_satellite","pattern":"The website has been blocked at this Public Wi-Fi (PWLAN) location because its content violates the PWLAN content filter rules\\.</span>"}
+{"fingerprint":"e_unk_redirect_localhost_satellite","pattern":"Location: http://127\\.0\\.0\\.1"}
+{"fingerprint":"e_unk_ru_451_1_satellite","pattern":"Как очистить кеш браузера вы можете посмотреть в <a class=\"link\" href=\"https://www\\.skydns\\.ru/guide/\">инструкции</a>"}
+{"fingerprint":"e_unk_ru_451_2_satellite","pattern":"Если вы не желаете видеть рекламу - перейдите на <a class=\"%!s(BADINDEX)\" href=\"https://www\\.skydns\\.ru/renew/\">"}
+{"fingerprint":"e_unk_ua_kyivstar_redirecr_302_1_satellite","pattern":"Location: https://cdn\\.kyivstar\\.ua/sites/default/files/lock_sites/index\\.html"}
+{"fingerprint":"f_gen_451_status_code_satellite","pattern":"451 Unavailable For Legal Reasons"}
```

Comment:
I think this points to geoblocking rather than censorsip. @ohnorobo @rensafi Should be consider this as a valid blockpage? We could even start a new code `g_<name>`  for geoblocking codes.